README.md

```diff
@@ -15,6 +16,11 @@ THE CODE IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND. HMS DOES NOT WARRANT
 
 A library to get historical or real time tag values.
 
+### [Memory Heap Size Notice](#table-of-contents)
+
+The operations performed in the historical data portions of this library use a significant amount of memory, and it is recommended that the Ewon Flexy Java heap size be increased to 20M or greater.
```

Comment:
20MB?

Reply:
Updated to also include the `xxMB` notation as well, although the `xxM` notation should still be preferred because that is how it appears in the `jvmrun` file.